solutions/simple/variables.tf

```diff
@@ -0,0 +1,400 @@
+            port_max        = 22
+            source_port_min = 1024
+            source_port_max = 65535
+          }
+          destination = "0.0.0.0/0"
+          source      = "0.0.0.0/0"
+        },
+        {
+          name      = "allow-all-443-outbound"
+          action    = "allow"
+          direction = "outbound"
+          tcp = {
+            source_port_min = 443
+            source_port_max = 443
+            port_min        = 1024
+            port_max        = 65535
+          }
+          destination = "0.0.0.0/0"
+          source      = "0.0.0.0/0"
+        },
+        {
+          name      = "allow-all-80-outbound"
+          action    = "allow"
+          direction = "outbound"
+          tcp = {
+            source_port_min = 80
+            source_port_max = 80
+            port_min        = 1024
+            port_max        = 65535
+          }
+          destination = "0.0.0.0/0"
+          source      = "0.0.0.0/0"
+        },
+        {
+          name      = "allow-all-22-outbound"
+          action    = "allow"
+          direction = "outbound"
+          tcp = {
+            source_port_min = 22
+            source_port_max = 22
+            port_min        = 1024
+            port_max        = 65535
+          }
+          destination = "0.0.0.0/0"
+          source      = "0.0.0.0/0"
+        }
+      ]
+    }
+  ]
+
+  validation {
+    error_message = "ACL rule actions can only be `allow` or `deny`."
+    condition = length(distinct(
+      flatten([
+        # Check through rules
+        for rule in flatten([var.network_acls[*].rules]) :
+        # Return false action is not valid
+        false if !contains(["allow", "deny"], rule.action)
+      ])
+    )) == 0
+  }
+
+  validation {
+    error_message = "ACL rule direction can only be `inbound` or `outbound`."
+    condition = length(distinct(
+      flatten([
+        # Check through rules
+        for rule in flatten([var.network_acls[*].rules]) :
+        # Return false if direction is not valid
+        false if !contains(["inbound", "outbound"], rule.direction)
+      ])
+    )) == 0
+  }
+
+  validation {
+    error_message = "ACL rule names must match the regex pattern ^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$."
+    condition = length(distinct(
+      flatten([
+        # Check through rules
+        for rule in flatten([var.network_acls[*].rules]) :
+        # Return false if direction is not valid
+        false if !can(regex("^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$", rule.name))
+      ])
+    )) == 0
+  }
+
+}
+
+
+##############################################################################
+# VPC Flow Logs Variables
+##############################################################################
+
+variable "enable_vpc_flow_logs" {
+  description = "Flag to enable vpc flow logs. If true, flow log collector will be created"
+  type        = bool
+  default     = false
+}
+
+variable "create_authorization_policy_vpc_to_cos" {
```

Comment:
keep consistent with other DAs - use the `skip_` naming convention

Reply:
done